website_sale_product_minimal_price/controllers/main.py

```diff
@@ -16,48 +16,33 @@ def get_combination_info_pricelist_atributes(
         """Special route to use website logic in get_combination_info override.
         This route is called in JS by appending _website to the base route.
         """
-        actual_qty = int(actual_qty)
         product = request.env["product.product"].browse(product_id)
-        current_website = request.env["website"].get_current_website()
-        today = fields.Date.today()
+        pricelist = request.env["website"].get_current_website().get_current_pricelist()
+        # Getting all min_quantity of the current product to compute the posible
+        # price scale.
+        qty_list = request.env["product.pricelist.item"].search([
+            "|",
+            ("product_id", "=", product.id),
+            "|",
+            ("product_tmpl_id", "=", product.product_tmpl_id.id),
+            ("categ_id", "in", list(map(int, product.categ_id.parent_path.split('/')[0:-1]))),
+            ("min_quantity", ">", 0),
+        ]).mapped("min_quantity")
+        qty_list = sorted(set(qty_list))
```

Comment:
How do you know that the pricelist is chained to the used one?

Reply:
In this case we take all the pricelist items asociated to this product to know what are the `minimal_qty`s that have to be checked, so we will know the scaled prices. This method is used just on the grid product price scale